os/mm/mm_heap/mm_calloc.c

```diff
@@ -81,12 +81,25 @@ FAR void *mm_calloc(FAR struct mm_heap_s *heap, size_t n, size_t elem_size)
 #endif
 {
 	FAR void *ret = NULL;
+	size_t size;
 
 	if (n > 0 && elem_size > 0) {
+		if (n > (MMSIZE_MAX / elem_size)) {
+			mdbg("Parameter n(%u) should be smaller than %u, \
+				because multiplication of n and elem_size cannot overflow the size_t.\n", n, elem_size);
+			return NULL;
+		}
+		size = n * elem_size;
```

Comment:
if state in line 86 is not true, size is not used. svace will treat this as minor issue.